packages/kit/test/nodash/internal.test.ts

```diff
@@ -91,4 +91,17 @@ describe('nodash internal', () => {
       expect(_.upperFirst('abc')).to.equal('Abc');
     });
   });
+
+  describe('format', () => {
+    it('returns a title case string given a camel case one', () => {
+      let result = _.camelCaseToTitleCase('FooBar');
+      expect(result).to.equal('Foo Bar');
+
+      result = _.camelCaseToTitleCase('fooBar');
+      expect(result).to.equal('Foo Bar');
+
+      result = _.camelCaseToTitleCase('Foo Bar');
+      expect(result).not.to.equal('Foo Bar');
```

Comment:
@maggiben Why is this test `not.equal` This seems to cover the case I mentioned above, "what happens if the string is already formatted in title case?"

But I would think that calling camelCaseToTitleCase with "Foo Bar" would return "Foo Bar".

Maybe test if the input string is properly formatted before transformation. If true then just return the input string.

Reply:
@peternhale 

But I would think that calling camelCaseToTitleCase with "Foo Bar" would return "Foo Bar".

In reality it returns "Foo  Bar" with one extra space added in between the two words:

![image](https://user-images.githubusercontent.com/2845947/112649977-b536c180-8e29-11eb-9a7e-bd6a3225a1a2.png)

I copy the source from here: https://github.com/salesforcecli/plugin-org/blob/6f0d1c4093056b09e2c9fdf5501f83cdd8e9fa1e/src/shared/utils.ts#L17 didn't want to change it but if you need it to be so it strips the extra spaces I can do that.

Reply:
@maggiben Please make the change to ensure that resulting string has only one space between words.

Reply:
@peternhale done